modules/ROOT/pages/performance-and-impact.adoc

```diff
@@ -26,14 +34,15 @@ Application metrics::
 30 days.
 
 Enhanced API metrics::
-30 days. The limit for data in the *Client Applications* tab is 1 hour.
+30 days. Data in the *Client Applications* tab is limited to 1 hour.
```

Comment:
Should it be one hour? (throughout)

Reply:
oops, I merged too soon. 😬 Will address this style issue later.